src/applications/vaos/services/appointment/transformers.js

```diff
@@ -358,159 +358,48 @@ function setLocation(appt) {
       return {
         vistaId: appt.facilityId,
         clinicId: appt.clinicId,
-        stationId: appt.sta6aid,
-        displayName:
+        stationId: appt.sta6aid || appt.facilityId,
```

Comment:
In some cases we don't have `sta6aid`, falling back to `facilityId` is generally ok.